tcms/rpc/tests/test_auth.py

```diff
@@ -0,0 +1,70 @@
+# -*- coding: utf-8 -*-
+# pylint: disable=attribute-defined-outside-init
+
+from xmlrpc.client import Fault as XmlRPCFault
+from django.contrib.sessions.models import Session
+from tcms.rpc.api.user import User
+from tcms.rpc.tests.utils import APITestCase
+from tcms.tests import UserFactory
+
+
+class TestAuthLogin(APITestCase):
+    """Test Auth.login method"""
+
+    def _fixture_setup(self):
+        super()._fixture_setup()
+        #
+        self.rpc_client.Auth.logout()
+        self.tester = UserFactory()
+        self.password = "password"
+        self.tester.set_password(self.password)
+        self.tester.save()
```

Comment:
In APITestCase you already have a user created which has a well-known password. You don't have to create your own.